pkg/controller/config-controller_test.go

```diff
@@ -636,6 +658,9 @@ func createConfigReconciler(objects ...runtime.Object) (*CDIConfigReconciler, *c
 	cdi := &cdiv1.CDI{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: "cdi",
+			Annotations: map[string]string{
+				"cdi.kubevirt.io/configAuthority": "",
```

Comment:
You made the const, you could use it here?

Reply:
Yeah, but I think it is better for tests to have their own copy of these consts that way you catch any changes which may have external impact.